src/components/buttons/button.tsx

```diff
@@ -13,9 +14,22 @@ interface ButtonProps {
   paddingHorizontal?: number;
   onClick?: () => void;
   expand?: boolean;
+  weight: 'bold' | 'regular' | 'semiBold';
```

Comment:
o no da prop parece confusa, não sei o que seria um botao com peso bold 🤔 
Poderia deixar claro que é uma prop pro titulo do botao:

```suggestion
  titleWeight?: 'bold' | 'regular' | 'semiBold';
```
outra coisa é definir ela como nullable, permitindo que receba um falor default `regular`, por exemplo, e vc tendo que mudar apensa quando variar